packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -1010,10 +1010,10 @@ function formatStakeDetails(db: DB, market: MarketData, stakeDetails: DisputeDoc
   for (let i = 0; i < stakeDetails.length; i++) {
     const outcomeDetails = stakeDetails[i];
     const outcomeValue = getOutcomeValue(market, outcomeDetails.payoutNumerators);
-    const bondSizeCurrent = new BigNumber(market.totalRepStakedInMarket, 16)
-    .multipliedBy(2)
-    .minus(new BigNumber(outcomeDetails.totalRepStakedInPayout || '0').multipliedBy(3)).toFixed();
-if (outcomeDetails.disputeRound < market.disputeRound) {
+    if (outcomeDetails.disputeRound < market.disputeRound) {
```

Comment:
Is this a revert of my earlier PR?

Reply:
Must have come from merging in master, will undo